br-ext/package/optee_client_ext/optee_client_ext.mk

```diff
@@ -2,6 +2,8 @@ OPTEE_CLIENT_EXT_VERSION = 1.0
 OPTEE_CLIENT_EXT_SOURCE = local
 OPTEE_CLIENT_EXT_SITE = $(BR2_PACKAGE_OPTEE_CLIENT_EXT_SITE)
 OPTEE_CLIENT_EXT_SITE_METHOD = local
+OPTEE_CLIENT_EXT_DEPENDENCIES = host-pkgconf \
+	$(if $(BR2_PACKAGE_UTIL_LINUX_LIBS),util-linux-libs,util-linux)
```

Comment:
Could this be simplified to just `util-linux` perhaps? This pattern is used in two places in upstream Buildroot, and each time the commits mention circular dependencies which I think might not apply here: https://github.com/buildroot/buildroot/commit/eb058222596 and https://github.com/buildroot/buildroot/commit/e3c86f5c9e4.